lib/bulkrax.rb

```diff
@@ -1,20 +1,103 @@
-require "bulkrax/engine"
+require 'bulkrax/engine'
 
 module Bulkrax
   class << self
-    mattr_accessor :parsers, :system_identifier_field
+    mattr_accessor :parsers, :system_identifier_field, :default_field_mapping, :reserved_properties, :field_mappings
+
     self.parsers = [
-      { name: "OAI - Dublin Core", class_name: "Bulkrax::OaiDcParser", partial: 'oai_fields' },
-      { name: "OAI - Qualified Dublin Core", class_name: "Bulkrax::OaiQualifiedDcParser", partial: 'oai_fields' },
-      { name: "CSV - Comma Separated Values", class_name: "Bulkrax::CsvParser", partial: 'csv_fields' }
+      { name: 'OAI - Dublin Core', class_name: 'Bulkrax::OaiDcParser', partial: 'oai_fields' },
+      { name: 'OAI - Qualified Dublin Core', class_name: 'Bulkrax::OaiQualifiedDcParser', partial: 'oai_fields' },
+      { name: 'CSV - Comma Separated Values', class_name: 'Bulkrax::CsvParser', partial: 'csv_fields' }
     ]
 
-    self.system_identifier_field = :source
+    self.system_identifier_field = 'source'
+
+    # Hash of Generic field_mappings for use in the view
+    # There must be one field_mappings hash per view parial
+    # Based on Hyrax CoreMetadata && BasicMetadata
+    # Override at appliation level to change
+    self.field_mappings = {
+      "Bulkrax::OaiDcParser" => {
```

Comment:
this is really nitpicky, but you switch to single quotes above, but are still using double quotes here

Reply:
I like nitpicky.